src/pages/beheer/aanvragen/index.tsx

```diff
@@ -219,9 +218,9 @@ const CPRequestsPage: NextPage = () => {
 			</CPAdminLayout>
 
 			<ProcessRequestBlade
-				isOpen={selected !== null}
-				selected={visits?.items?.find((x) => x.id === selected)}
-				onClose={() => setSelected(null)}
+				isOpen={!!filters.visitRequest}
+				selected={visits?.items?.find((x) => x.id === filters.visitRequest)}
```

Comment:
You should handle the case where a visitRequest id is in the url, but the visitRequest does not exist:
![image](https://user-images.githubusercontent.com/1710840/165466430-566b101e-66ae-47a5-81c0-e314e6aa77bb.png)